docs/components/modeler/what-is-camunda-modeler.md

```diff
@@ -5,11 +5,11 @@ title: What is Camunda Modeler?
 
 Any executable process needs a **BPMN diagram** designed and configured beforehand.
 
-The design is used to specify the flow of a process. Thereby, different events must be configured in the model so the workflow engine understands what must executed.
+A BPMN diagram is used to visually outline the structure and flow of a process. As a result, the process can be more easily understood by various stakeholders.
 
-A configuration can be the conditions in a gateway or the specification of a service task.
+In tandem, different events and implementation details (such as the conditions within a gateway, or the specifications of a service task) must be configured in the model so the workflow engine understands what must executed once the process reaches a certain task.
 
-Camunda offers three tools to achieve this:
+Camunda offers three tools to implement these details:
```

Comment:
Core to all drawing tools is that they do both:

* draw (or design) your diagrams
* implement them (adding implementation details)

Let us phrase it this way. The tools are not meant to be "implementation details adders".

Reply:
Updated in latest commit. Thoughts?